Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM bitnami/minideb AS builder
+RUN install_packages python-pip curl tar python-setuptools rsync binutils
+RUN pip install dockerize
+RUN mkdir -p /workdir
+WORKDIR /workdir
+ENV DOCKER_TAG v0.7.0
+RUN curl -L https://github.com/getzola/zola/releases/download/$DOCKER_TAG/zola-$DOCKER_TAG-x86_64-unknown-linux-gnu.tar.gz | tar xz
```

Comment:
how does that work with the dockerhub automatic build?

Reply:
1. Create a repo on dockerhub and connect your github account.
2.  Go to the build settings (easy way, append /builds/edit to the dockerhub repository url)
3. Use this settings 
![cloud docker com_repository_docker_leiserfg_zola_builds_edit](https://user-images.githubusercontent.com/2947276/58687713-b2c8c500-8382-11e9-958a-6acd6de522b8.png)

Reply:
But in that dockerfile the version is hardcoded?